### PR TITLE
Unifyed task names to runtime task names

### PR DESCRIPTION
**Pull request by Shigeyuki-fukuda**
## Problem
The generated task name is not consistent with the runtime task name.
Therefore, I don't think it can be done intuitively.

### For example

generated task name： `20200530_foo_bar.rake`

```console
$ bin/rails generate oneshot foo_bar
Running via Spring preloader in process 93693
      create  lib/tasks/oneshot/20200530_foo_bar.rake
```

runtime task name： `foo_bar_20200530`

```console
$ bin/rake oneshot:foo_bar_20200530
```

## Solution
I unifyed task names to runtime task names.

### For example

generated task name： `foo_bar_20200530.rake`

```console
$ bin/rails generate oneshot foo_bar
Running via Spring preloader in process 93857
      create  lib/tasks/oneshot/foo_bar_20200530.rake
```

runtime task name： `foo_bar_20200530`

```console
$ bin/rake oneshot:foo_bar_20200530
```

## Spec
I checked the file generated by the modified source and modified the RSpec.

### CODE REVIEW

**Comment by s-osa** [comment]
Thank you for the pull request 😄 

I completely agree with the importance of consistency. But the date prefix of filenames is also a vital interface to glance files.

How about changing task names instead of filenames?

**Comment by Shigeyuki-fukuda** [comment]
@s-osa 

Thank you for your reply.
> How about changing task names instead of filenames?

I also agree with that opinion:ok_hand:
I changed my policy in https://github.com/s-osa/oneshot_task_generator/pull/13/commits/b4ef8547feba9007469e4277bf3199e7115d4aff .

**Comment by s-osa** [comment]
The change is LGTM. Thanks a lot.

Could you summarize changes into one commit?
After you squash the commits, I'd like to merge and release the new version.

**Comment by Shigeyuki-fukuda** [comment]
@s-osa 

>The change is LGTM. Thanks a lot.

Thank you, too 😄  .



>Could you summarize changes into one commit?
After you squash the commits, I'd like to merge and release the new version.

I summarized changes into one commit.
https://github.com/s-osa/oneshot_task_generator/pull/13/commits/19d37560e51d51b972e8f4c896009eda9f4b5ac2

**Review by s-osa — APPROVED** [review]

**Comment by Shigeyuki-fukuda** [comment]
@s-osa 
Thank you for your review 🙏

**Comment by s-osa** [comment]
I've released the new version, including this change.  Please try it 😉 
https://rubygems.org/gems/oneshot_task_generator/versions/0.2.0
